Common/include/Luau/Bytecode.h

```diff
@@ -426,8 +426,8 @@ enum LuauBytecodeTag
 {
     // Bytecode version; runtime supports [MIN, MAX], compiler emits TARGET by default but may emit a higher version when flags are enabled
     LBC_VERSION_MIN = 3,
-    LBC_VERSION_MAX = 4,
-    LBC_VERSION_TARGET = 4,
+    LBC_VERSION_MAX = 5,
+    LBC_VERSION_TARGET = 5,
```

Comment:
TARGET has to be left at 4 until flag is removed (Roblox will take care of this future removal).

`BytecodeBuilder::getVersion()` should be updated to return 5 when flag is enabled.

Reply:
Fixed